packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -1,72 +1,72 @@
-module CTV = Reducer_Extension.CodeTreeValue
-module Lib = Reducer_Extension.ReducerLibrary
-module ME = Reducer_MathJs.Eval
-module Rerr = Reducer_Error
+module ExternalLibrary = ReducerInterface.ExternalLibrary
+module MathJs = Reducer_MathJs
+open ReducerInterface.ExpressionValue
+open Reducer_ErrorValue
+
 /*
   MathJs provides default implementations for builtins
   This is where all the expected builtins like + = * / sin cos log ln etc are handled
   DO NOT try to add external function mapping here!
 */
-type codeTreeValue = CTV.codeTreeValue
-type reducerError = Rerr.reducerError
 
 exception TestRescriptException
 
-let callInternal = (call: CTV.functionCall): result<'b, reducerError> =>{
-
-  let callMatjJs = (call: CTV.functionCall): result<'b, reducerError> =>
+let callInternal = (call: functionCall): result<'b, errorValue> => {
+  let callMathJs = (call: functionCall): result<'b, errorValue> =>
     switch call {
-      | ("jsraise", [msg]) => Js.Exn.raiseError(CTV.show(msg)) // For Tests
-      | ("resraise", _) => raise(TestRescriptException) // For Tests
-      | call => call->CTV.showFunctionCall-> ME.eval
+    | ("jsraise", [msg]) => Js.Exn.raiseError(show(msg)) // For Tests
+    | ("resraise", _) => raise(TestRescriptException) // For Tests
```

Comment:
Does this mean ``rescriptRaise``? I'd just call it that; I thought it might have been a misspelling for ``reraise``

Reply:
resraise is a fake LISP function to raise reScript exceptions for testing. The LISP source (resraise) results in a reScript expression. Then the source code (resraise) is used to test all Exception holes are plugged.

Reply:
Now we are entering into the good naming conventions for lisp source code...